### PR TITLE
EVA-520 Variant normalizer and loader steps using JobParameters

### PR DESCRIPTION
**Pull request by cyenyxe**
`VariantNormalizerStep` and `VariantLoaderStep` are now using Spring Batch native `JobParameters` instead of a custom "pipelineOptions". They are also injected in jobs following the same style.

Changes to test classes include:
- Providing jobs and steps with `JobParameters` objects
- Using an output temporary folder that is deleted after each test
- Test resources moved to subfolders when possible, copied when used by other non-ported tests

### CODE REVIEW

File: src/main/java/uk/ac/ebi/eva/pipeline/jobs/steps/VariantLoaderStep.java [hunk 1]
```diff
@@ -56,26 +54,19 @@
...
         URI transformedVariantsUri = outdirUri.resolve(outputVariantJsonFile.getFileName().toString());
 
-        logger.info("-- PreLoad variants -- {}", nextFileUri);
-        variantStorageManager.preLoad(transformedVariantsUri, outdirUri, variantOptions);
-        logger.info("-- Load variants -- {}", nextFileUri);
-        variantStorageManager.load(transformedVariantsUri, variantOptions);
-//          logger.info("-- PostLoad variants -- {}", nextFileUri);
-//          variantStorageManager.postLoad(transformedVariantsUri, outdirUri, variantOptions);
+        logger.info("Loading variants from file {}", inputFileUri);
+        variantStorageManager.load(transformedVariantsUri, jobOptions.getVariantOptions());
```
Comment (poggio84) [inline src/main/java/uk/ac/ebi/eva/pipeline/jobs/steps/VariantLoaderStep.java]: `jobOptions.getVariantOptions()` will be removed later?

Comment (cyenyxe) [inline src/main/java/uk/ac/ebi/eva/pipeline/jobs/steps/VariantLoaderStep.java]: Yes, that requires a lot more work that is still in progress.

File: src/test/java/uk/ac/ebi/eva/pipeline/jobs/GenotypedVcfJobTest.java [hunk 2]
```diff
@@ -75,9 +76,6 @@
  * and avoid NoUniqueBeanDefinitionException. There are also other solutions like:
  *  - http://stackoverflow.com/questions/29655796/how-can-i-qualify-an-autowired-setter-that-i-dont-own
  *  - https://jira.spring.io/browse/BATCH-2366
```
Comment (poggio84) [inline src/test/java/uk/ac/ebi/eva/pipeline/jobs/GenotypedVcfJobTest.java]: This is no longer valid and can be removed.

**Comment by cyenyxe** [comment]
Is there anything else preventing this from being merged?

**Pull request merged**